src/source/src/connector_source.rs

```diff
@@ -58,27 +58,19 @@ struct InnerConnectorSourceReader {
     context: SourceContext,
 }
 
-struct InnerConnectorSourceReaderHandle {
-    stop_tx: oneshot::Sender<()>,
-    join_handle: JoinHandle<()>,
-}
-
 /// [`ConnectorSource`] serves as a bridge between external components and streaming or
 /// batch processing. [`ConnectorSource`] introduces schema at this level while
-/// [`SplitReaderImpl`] simply loads raw content from message queue or file system.
+/// [`SplitReaderImpl`] simply loadsF raw content from message queue or file system.
 /// Parallel means that multiple [`InnerConnectorSourceReader`] will run in parallel during the
 /// `next`, so that 0 or more Splits reads can be handled at the Source level.
+#[allow(dead_code)]
```

Comment:
```suggestion
```

Reply:
Temporarily reserved

Reply:
Please clean it up before PR gets merged. :)

Reply:
> Please clean it up before PR gets merged. :)

removed